packages/css/src/components/ordered-list/ordered-list.scss

```
@@ -36,6 +36,12 @@
   }
 }
 
+.amsterdam-ordered-list--inverse-color,
+.amsterdam-ordered-list--inverse-color:not(.amsterdam-ordered-list--no-markers),
+.amsterdam-ordered-list--inverse-color .amsterdam-ordered-list {
```

Each instance of a component is responsible for its own appearance. Nested lists should have the `inverseColor` prop explicitly as well.

```suggestion
.amsterdam-ordered-list--inverse-color:not(.amsterdam-ordered-list--no-markers) {
```